content/vehicles/cv-9040a.md

```
@@ -6,8 +6,12 @@ frontArmorDepth: 60
 
 ## Description
 
-The **CV 9040A** is an Infantry Fighting Vehicle and a Swedish tracked APC/IFV armed with a 40 mm cannon. The CV 90 family entered service in the 1990s and is used by Sweden and several export customers; the 9040 variant emphasizes anti-air and anti-materiel capability. The 23.5 t vehicle carries a crew of three plus six passengers and reaches 70 km/h forward and 45 km/h reverse. The **Scania DSI14** diesel produces 550 hp for 23.4 hp/t and an automatic gearbox.\
-The **40mm Akan m/70B** is in a stabilized turret with a 24-round magazine and is paired with a **7.62mm ksp 39 C**. The gunner has a laser rangefinder, Gen 2 thermals, 8x zoom, and FCS with lead; the vehicle has smoke grenades but no horizontal stabilizer for the main gun.
+The **CV 9040A** is an Infantry Fighting Vehicle and a Swedish tracked APC/IFV, whose Swedish designation is **Strf 9040A**. It is armed with a 40 mm cannon. The CV 90 family entered service in the 1990s and is used by Sweden and several export customers; the 9040 variant emphasizes anti-air and anti-materiel capability. The 23.5 t vehicle carries a crew of three plus six passengers and reaches 70 km/h forward and 45 km/h reverse. The **Scania DSI14** diesel produces 550 hp for 23.4 hp/t and an automatic gearbox.\
+The **40mm Akan m/70B** has a 24-round magazine and is paired with a **7.62mm ksp 39 C**. The gunner has a laser rangefinder, Gen 2 thermals, 8x zoom, and FCS with lead; the vehicle has smoke grenades but no horizontal stabilizer for the main gun.
+The vehicle also features 3 modifications: CV 9040 BILL, CV 9040B and slpprj m/01, which can all be used together.
+The CV 9040 BILL adds the RBS 56 ATGM launcher to the vehicle; this increases the CV 9040A's overall anti-tank capabilities, however, to fire the ATGM you also need an additional crew member. 
+The CV 9040B modification updates the turret of the CV 9040, giving the gun a stabiliser and adding a backup monochromatic sight to the vehicle. 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)